### PR TITLE
Streamline the existing PHPDoc comments

### PR DESCRIPTION
**Pull request by oliverklee**
Make them more useful for static code analysis, for human readers
and for rendering.

### CODE REVIEW

File: src/Comment/Comment.php [hunk 1]
```diff
@@ -33,7 +33,7 @@ public function getLineNo()
     }
 
     /**
-     * @return string
+     * @paran string $sComment
```
Comment (westonruter) [inline src/Comment/Comment.php:36]:
```suggestion
     * @param string $sComment
```

**Review by westonruter — CHANGES_REQUESTED** [review]

File: src/CSSList/CSSList.php [hunk 2]
```diff
@@ -281,7 +281,7 @@ public function remove($oItemToRemove)
     /**
      * Replaces an item from the CSS list.
      *
-     * @param RuleSet|Import|Charset|CSSList $oItemToRemove
+     * @param RuleSet|Import|Charset|CSSList $oOldItem
      *        May be a RuleSet (most likely a DeclarationBlock), a Import, a Charset
      *        or another CSSList (most likely a MediaQuery)
```
Comment (westonruter) [inline src/CSSList/CSSList.php:286]: Shouldn't this also add a `@param` for `$mNewItem?`

Comment (oliverklee) [reply src/CSSList/CSSList.php:286]: Sort of :-) … in this PR, I would like to mostly update existing PHPDoc comments. As there are lots of type annotations missing, I plan to add them in several dedicated PRs.

**Review by westonruter — APPROVED** [review]

**Comment by oliverklee** [comment]
I have added a second commit that changes `@-rule` to `at-rule` as the `@` character marks PHPDoc annotations.

**Comment by oliverklee** [comment]
I'll take care of the rebase in order to resolve the merge conflicts.

**Comment by oliverklee** [comment]
Rebased on top of the latest changes.

**Pull request merged**